docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Button.htm

```diff
@@ -484,18 +489,18 @@ <h3>Summary</h3>
 						<div class="example">
 							<span class="example"><p>Code
 								example:</p><p></p></span>
-							<pre name="code" class="examplecode
-							prettyprint">
-&lt;div class=&quot;ui-btn&quot; id=&quot;button&quot;&gt;&lt;/div&gt;
-&lt;script&gt;
-   var element = document.getElementById(&quot;button&quot;),
-       buttonWidget = tau.widget.Button(element),
-       /* Value contains inner text of button */
-       value = buttonWidget.value();
-
-   /* &quot;New text&quot; will be text of button */
-   buttonWidget.value(&quot;New text&quot;);
-&lt;/script&gt;
+<p>HTML code:</p>
+<pre class="prettyprint">&lt;div class=&quot;ui-btn&quot; id=&quot;button&quot;&gt;&lt;/div&gt;</pre>
+<p>JS code:</p>
+<pre class="prettyprint">
+var element = document.getElementById(&quot;button&quot;),
+    buttonWidget = tau.widget.Button(element),
+
+/* Value contains inner text of button */
```

Comment:
Would you please check the indentation of line 499 and 500?

Reply:
Fixed